### PR TITLE
Adding info for scripts

### PR DESCRIPTION
**Pull request by mrbodean**
These changes may be more appropriate as the start of a security page for scripts.
Added a best pratice statement for allowing script authors to approve their own scripts. 
Fixed a typo for bolding.
Added note for the context that scripts are excuted in

### CODE REVIEW

**Comment by arob98** [comment]
@mattbriggs For your review.

File: sccm/apps/deploy-use/create-deploy-scripts.md [hunk 1]
```diff
@@ -57,9 +57,12 @@ By default, users cannot approve a script they have authored. Because scripts ar
 3. In the list of sites, choose your site and then, on the **Home** tab, in the **Sites** group, click **Hierarchy Settings**.
 4. On the **General** tab of the **Hierarchy Settings Properties** dialog box, clear the checkbox **Do not allow script authors to approve their own scripts**.
 
+>[!IMPORTANT]
+>As a Best Practice you should not allow a script author to approve their own scripts! This should only be allowed in a Lab setting. Please carefully consider the potential impact of changing this setting in a Production environment.
```
Comment (mattbriggs) [inline sccm/apps/deploy-use/create-deploy-scripts.md]: +>[!IMPORTANT] 
+>As a best practice, you shouldn't allow a script author to approve their own scripts. This should only be allowed in a lab setting. Please carefully consider the potential impact of changing this setting in a production environment.

Comment (mattbriggs) [reply sccm/apps/deploy-use/create-deploy-scripts.md]: just changing the caps... :)

File: sccm/apps/deploy-use/create-deploy-scripts.md [hunk 2]
```diff
@@ -107,6 +110,9 @@ After a script is approved, it can be run against a collection you choose.
 >[!IMPORTANT]
 >The script is given a one-hour time period in which to run. If it does not run (for example if the PC is turned off) in this time period, you must run it again.
 
+>[!IMPORTANT]
+>The script is executed as the system or computer account on the targeted client(s). This account has very limited network access. Any access to remote systems and\or locations by the script must be provisioned with this in mind.
```
Comment (mattbriggs) [inline sccm/apps/deploy-use/create-deploy-scripts.md]: > [!IMPORTANT] 
> The script is executed as the system or computer account on the targeted client(s). This account has very limited network access. Any access to remote systems and locations by the script must be provisioned with this in mind.

Comment (mattbriggs) [reply sccm/apps/deploy-use/create-deploy-scripts.md]: Thanks ... "and\or" are difficult for localization.

Comment (mrbodean) [reply sccm/apps/deploy-use/create-deploy-scripts.md]: Thanks @mattbriggs

**Comment by mattbriggs** [comment]
@mrbodean Hi, would you mind making your updates, and I can submit the merge? Thanks again for this. Cheers! Matt

**Pull request merged**